README.md

```diff
@@ -88,7 +114,6 @@ The Outline Client uses a **generated mobile library** on Android, iOS and macOS
 
 Below we provide more details on each integration approach. For more details about setting up and using Outline SDK features, see the [Discussions tab](https://github.com/Jigsaw-Code/outline-sdk/discussions).
 
-
```

Comment:
It seems there is a convention of using two empty lines before section titles. No need to remove this.